core/channels.config.ts

```diff
@@ -0,0 +1,24 @@
+const DEFAULT_CHANNEL_ID = process.env.BIGCOMMERCE_CHANNEL_ID;
+
+export default {
+  en: DEFAULT_CHANNEL_ID,
+  da: DEFAULT_CHANNEL_ID,
+  de: DEFAULT_CHANNEL_ID,
+  'es-419': DEFAULT_CHANNEL_ID,
+  'es-AR': DEFAULT_CHANNEL_ID,
+  'es-CL': DEFAULT_CHANNEL_ID,
+  'es-CO': DEFAULT_CHANNEL_ID,
+  'es-LA': DEFAULT_CHANNEL_ID,
+  'es-MX': DEFAULT_CHANNEL_ID,
+  'es-PE': DEFAULT_CHANNEL_ID,
+  es: DEFAULT_CHANNEL_ID,
+  fr: DEFAULT_CHANNEL_ID,
+  it: DEFAULT_CHANNEL_ID,
+  ja: DEFAULT_CHANNEL_ID,
+  nl: DEFAULT_CHANNEL_ID,
+  no: DEFAULT_CHANNEL_ID,
+  pl: DEFAULT_CHANNEL_ID,
+  'pt-BR': DEFAULT_CHANNEL_ID,
+  pt: DEFAULT_CHANNEL_ID,
+  sv: DEFAULT_CHANNEL_ID,
+};
```

Comment:
@jorgemoya does it make sense to combine this with locales from i18n into one config to minimize number of places where they're hardcoded?

Reply:
I have updated this object to just include the overrides, if any. I have made some type changes to make sure the keys match any of our configured locales. With these changes I don't think it makes sense to merge together, what do you think?

Reply:
Great idea!